content/member/rledru.yaml

```
@@ -0,0 +1,30 @@
+active: true
+🚲: true
+
+name: Romane Ledru
+gender: female
+position: "Alternante en développement"
+bio: "Romane s'est passionnée pour le web après s'être longuement aventurée dans le domaine de l'Arts. Devant une feuille blanche comme devant du code, la création reste son grand dada.
```

❔ **question:** Arts au pluriel, c'est souhaité dans ce contexte précis ?

```suggestion
bio: "Romane s'est passionnée pour le web après s'être longuement aventurée dans le domaine de l'Art. Devant une feuille blanche comme devant du code, la création reste son grand dada.
```